src/tags/object/AudioNext/view.tsx

```diff
@@ -13,9 +14,15 @@ const AudioNextView: FC<AudioNextProps> = ({ item }) => {
   const [position, setPosition] = useState(1);
   const [audioLength, setAudioLength] = useState(0);
 
-  const [zoom, setZoom] = useState(Number(item.defaultzoom));
-  const [volume, setVolume] = useState(Number(item.defaultvolume));
-  const [speed, setSpeed] = useState(Number(item.defaultspeed));
+  const numberify = (val: any, defaults: Record<string, number>) => {
```

Comment:
this can be lifted out of the component definition as it is a pure function.

Reply:
done